src/test/java/com/angelbroking/smartapi/smartstream/SmartStreamListenerImpl.java

```diff
@@ -105,6 +106,16 @@ public void onPong() {
 		log.info("pong received");
 	}
 
+	/* Custom on error method for SmartStream */
+	@Override
+	public SmartStreamError onErrorCustom() {
+		// User thrown error return
+		log.info("on custom error");
+		SmartStreamError smartStreamError = new SmartStreamError();
+		smartStreamError.setException(new SmartAPIException("custom error received"));
```

Comment:
what does it mean by custom error received?

Reply:
it is just a test message, user can write its own message like "Exception occured".